sdk/aqueduct/integrations/sql_integration.py

```diff
@@ -127,14 +145,43 @@ def sql(
                 query=extract_params,
             )
 
-        operator_id = generate_uuid()
-        output_artifact_id = generate_uuid()
+        # Find any tags that need to be expanded in the query, and add the parameters that correspond
+        # to these tags as inputs to this operator. The orchestration engine will perform the replacement at runtime.
+        sql_input_artifact_ids = []
+        if extract_params.query is not None:
+            matches = re.findall(TAG_PATTERN, extract_params.query)
+            for match in matches:
+                param_name = match.strip(" {}")
+
+                # If it is a built-in tag, we can ignore it for now, since the python operators will perform the expansion.
+                if param_name in BUILT_IN_EXPANSIONS:
```

Comment:
what happens if the user is 'overriding' the built-in name? Should we validate the value if there's an user defined params op exists?

Reply:
You're right. The backend will first check for user-defined expansion before built-in expansions, so this line should go within the `if param_op is None` check (we should only continue if its a built-in expansion but there doesn't exist a parameter for it)